api/subscriptions/serializers.py

```diff
@@ -1,58 +1,43 @@
 from rest_framework import serializers as ser
```

Comment:
Changes here are to make the new model backwards compatible with the old view, so basically the notification type is being converted into a legacy `event_name` and `_id` value.